bitnami/kubeapps/values.yaml

```diff
@@ -1974,27 +1974,22 @@ testImage:
 ## PostgreSQL chart configuration
 ## ref: https://github.com/bitnami/charts/blob/master/bitnami/postgresql/values.yaml
 ## @param postgresql.enabled Deploy a PostgreSQL server to satisfy the applications database requirements
-## @param postgresql.architecture PostgreSQL architecture (`standalone` or `replication`)
 ## @param postgresql.auth.postgresPassword Password for 'postgres' user
 ## ref: https://github.com/bitnami/bitnami-docker-postgresql/blob/master/README.md#setting-the-root-password-on-first-run
 ## @param postgresql.auth.database Name for a custom database to create
 ## @param postgresql.auth.existingSecret Name of existing secret to use for PostgreSQL credentials
 ##
 postgresql:
   enabled: true
-  architecture: "replication"
```

Comment:
Why remove it instead of using `standalone` ? I mean, just to make it more explicit and let users override the value more easily (because if present, it gets documented in the chart values)

Reply:
Sorry, this PR was landed before I got a chance to revert/reply.

I removed it simply because "standalone" was the default value and i assumed we only had the `architecture` present in our kubeapps values because we were overriding the default value, so now that we weren't overriding it I didn't see an issue removing it. More generally, I think the values of the subchart that we explicitly call out in our values.yaml are a combination of those we override and those that make sense to present for different reasons: requirement (`postgresPassword`), importance (perhaps `architecture`?). Happy to revert and re-include this one if it's an important one.